pyface/ui/wx/progress_dialog.py

```diff
@@ -46,20 +47,39 @@ def __init__(
         maximum=100,
         direction="horizontal",
         size=(200, -1),
+        **traits,
     ):
         """
         Constructs a progress bar which can be put into a panel, or optionaly,
         its own window
 
         """
-        self._max = maximum
-        self.parent = parent
+        create = traits.pop("create", True)
+
+        # XXX minimum is ignored - it either should be deprecated or supported
+        super().__init__(
+            parent=parent,
+            _max=maximum,
+            direction=direction,
+            size=size,
```

Comment:
looks like `size` is ignored as well - it isn't defined on the `IWidget` interface or on the wx `Widget` class

Reply:
Issue created for this and `minimum`: #996